### PR TITLE
solution

### PR DESCRIPTION
**Pull request by NikitaNikolaitsev**

### CODE REVIEW

File: cinema/views.py [hunk 1]
```diff
@@ -1,45 +1,94 @@
...
+    def get(self, request: Request) -> Response:
+        genres = Genre.objects.all()
+        serializer = GenreSerializer(genres, many=True)
         return Response(serializer.data, status=status.HTTP_200_OK)
 
-    if request.method == "POST":
-        serializer = MovieSerializer(data=request.data)
+    def post(self, request: Request) -> Response:
+        serializer = GenreSerializer(data=request.data)
         if serializer.is_valid():
```
Comment (maxgerman) [inline cinema/views.py:38]: how can we avoid this "if" and save the same functionality?

**Review by maxgerman — CHANGES_REQUESTED** [review]
The tests don't pass :(

File: cinema/views.py [hunk 2]
```diff
@@ -1,45 +1,137 @@
...
+    ListModelMixin,
+    CreateModelMixin,
+    RetrieveModelMixin,
+    UpdateModelMixin,
+    DestroyModelMixin
+)
+from rest_framework.viewsets import GenericViewSet, ModelViewSet
 from django.shortcuts import get_object_or_404
+from rest_framework import status, mixins, generics, viewsets
+from rest_framework.request import Request
```
Comment (Arsen-hrynevych) [inline cinema/views.py:12]: Imports should be grouped and separated by a blank line according to standard library, external libraries, and custom imports. This practice enhances both code readability and maintainability. 
Read more about topic [here](https://peps.python.org/pep-0008/#imports)

**Review by Arsen-hrynevych — APPROVED** [review]
Overall good. Thanks

**Pull request closed without merging**